.cirrus.yml

```diff
@@ -311,6 +317,8 @@ macos_aarch64_min_version_debug_task:
   <<: *IOX2_COMMON_BUILD_AND_TEST_DEBUG
 
 macos_aarch64_stable_debug_task:
+  # TODO commented out due to limited CI time
+  only_if: false
```

Comment:
I'm a bit torn on this. On one hand macOS takes quite a lot of CI time but on the other hand it is another aarch64 target which could help finding issues with lock-free code